markdownlint.js

```diff
@@ -161,7 +161,37 @@ function printResult(lintResult) {
     process.exitCode = 1;
   }
 
-  if (program.output) {
+  if (program.junit) {
+    const builder = require('junit-report-builder');
+    const suite = builder
```

Comment:
Mind calling it testSuite?

Reply:
I figured there'd be little stylistic things here I wasn't aware of, changed